tests/test_subdirectory.py

```diff
@@ -209,13 +227,24 @@ def test_new_version_uses_subdirectory(tmp_path_factory):
         git("tag", "v2")
 
     # Finally, update the generated project
-    copier.copy(dst_path=project_path, defaults=True, overwrite=True)
+    copier.copy(dst_path=project_path, defaults=True, overwrite=True, conflict=conflict)
     assert "_commit: v2" in (project_path / ".copier-answers.yml").read_text()
 
-    # Assert that the README still exists, and was force updated to "upstream version 2"
+    # Assert that the README still exists, and the conflicts were handled
+    # correctly.
     assert (project_path / "README.md").exists()
-    with (project_path / "README.md").open() as fd:
-        assert fd.read() == "upstream version 2\n"
+
+    # On Windows, skip checking the inline conflicts due to line ending issues.
+    if conflict == "rej" or os.name != "nt":
```

Comment:
Please use an appropriate xfail mark: https://github.com/copier-org/copier/blob/080e3f645e26c01ccce314634159651001dcf1cd/tests/test_migrations.py#L25-L29

Reply:
Will do. It will look a bit different because it's 1 of 2 cases failing for a parametrized test.

I followed the suggestion here?
https://stackoverflow.com/questions/30571481/how-to-mark-test-as-xfail-only-with-specific-parameters-with-pytest